Daylight.xcodeproj/project.pbxproj

```diff
@@ -496,7 +507,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "/usr/local/bin/carthage copy-frameworks";
+			shellScript = "/usr/local/bin/carthage copy-frameworks\n";
```

Comment:
any reason this got a newline?

Reply:
🤔

Reply:
I removed it, but if i open the Build Phases it get's added automatically 🤷‍♂️

Reply:
So i guess it's better to leave it...

Reply:
Yep, leave it 👍